var/spack/repos/builtin/packages/py-geopandas/package.py

```diff
@@ -48,15 +50,21 @@ class PyGeopandas(PythonPackage):
     depends_on("py-pandas@0.25.0:", type=("build", "run"), when="@0.10:")
     depends_on("py-pandas@1.0.0:", type=("build", "run"), when="@0.11:")
     depends_on("py-pandas@1.4.0:", type=("build", "run"), when="@0.14:")
-    depends_on("py-shapely@:1", type=("build", "run"))
+    depends_on("py-shapely@:1", type=("build", "run"), when="@:0.99")
     depends_on("py-shapely@1.6:1", type=("build", "run"), when="@0.9:0.10")
-    depends_on("py-shapely@1.7:1", type=("build", "run"), when="@0.11:")
+    depends_on("py-shapely@1.7:1", type=("build", "run"), when="@0.11:0.99")
     depends_on("py-shapely@1.8.0:", type=("build", "run"), when="@0.14:")
-    depends_on("py-fiona", type=("build", "run"))
-    depends_on("py-fiona@1.8:", type=("build", "run"), when="@0.9:")
-    depends_on("py-fiona@1.8.21:", type=("build", "run"), when="@0.14:")
+    depends_on("py-shapely@2.0.0:", type=("build", "run"), when="@1:")
     depends_on("py-pyproj", type=("build", "run"))
     depends_on("py-pyproj@2.2.0:", type=("build", "run"), when="@0.7:")
     depends_on("py-pyproj@2.6.1.post1:", type=("build", "run"), when="@0.11:")
     depends_on("py-pyproj@3.3.0:", type=("build", "run"), when="@0.14:")
     depends_on("py-packaging", type=("build", "run"), when="@0.11:")
+    # Only for versions 0.14.4+ ? Or was it forgotten in pyproject.toml for 0.14.0 to 0.14.3?
```

Comment:
Good catch, geopandas has always had a dependency on numpy (even back in 0.3.0) but it was never documented. I would put a generic `depends_on("py-numpy")` for all versions in addition to what you already have.

Reply:
done